src/components/bulk-email-tool/bulk-email-form/data/__factories__/bulkEmailFormCourseMode.factory.js

```diff
@@ -0,0 +1,18 @@
+import { Factory } from 'rosie'; // eslint-disable-line import/no-extraneous-dependencies
+
+const courseModeFactory = () => {
```

Comment:
It would be a good idea to add some [jsDocs](https://jsdoc.app/) documentation for this code. What do you think?

Reply:
OK. It has already been added. Check please and add any suggestions if needed.